pkg/utils/kubernetes/crd.go

```diff
@@ -89,7 +89,7 @@ var (
 	initOnceFn sync.Once
```

Comment:
Is this still needed?

Reply:
This was removed in https://github.com/gardener/gardener/pull/11545/commits/6dc30ab5771f21b85f76dbc8f54943128ad196d3